app/views/govuk_component/docs/search.yml

```diff
@@ -1,19 +1,22 @@
 name: "Search"
 description: "Search box"
+body: |
+  Component built primarily for the GOV.UK homepage (where it uses the on_dark_background option) but intended for use throughout the site.
+
+  Component provides only the search box itself and must be wrapped in a form.
```

Comment:
Here's what I think the body should say:

> A search box with label and  attached submit button. The component must be used within an HTML form.
> The search input has a `name="q"` attribute and a customisable ID.
>
> It can be used on white or on `govuk-blue` using the `on_dark_background` option.